Tasks/JenkinsDownloadArtifactsV1/task.loc.json

```diff
@@ -18,7 +18,7 @@
   "demands": [],
   "version": {
     "Major": 1,
-    "Minor": 175,
+    "Minor": 190,
```

Comment:
Please check if this minor version needs to be updated

Reply:
@chandrasekharmudili, yes, according to the [Bumping task version](https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/taskversionbumping.md) instruction we should update the minor version in both task.json, task.loc.json.